ui/src/app/app.module.ts

```diff
@@ -60,13 +61,15 @@ import {NumberRangeFilterComponent} from "./components/runs/filters/number-range
     StringFilterComponent,
     DatetimeRangeFilterComponent,
     StatusFilterComponent,
-    NumberRangeFilterComponent
+    NumberRangeFilterComponent,
+    MultipleStatusFilterComponent
   ],
   imports: [
     BrowserModule,
     AppRoutingModule,
     ClarityModule,
     FormsModule,
+    ReactiveFormsModule,
```

Comment:
Do you really need the `ReactiveFormsModule`?

Reply:
For this PR no we do not, maybe for workflow screen we might need it. For now, I have removed it.